File: common/protcol/c#/module/gm_centermodule.cs
/*this module file is codegen by juggle for c#*/
using System;
using System.Collections;
using System.Collections.Generic;

namespace module
{
    public class gm_center : juggle.Imodule 
    {
        public gm_center()
        {
			module_name = "gm_center";
        }

        public delegate void confirm_gmhandle(String argv0);
        public event confirm_gmhandle onconfirm_gm;
        public void confirm_gm(ArrayList _event)
        {
            if(onconfirm_gm != null)
            {
                var argv0 = ((String)_event[0]);
                onconfirm_gm( argv0);
            }
        }

        public delegate void close_clutterhandle(String argv0);
        public event close_clutterhandle onclose_clutter;
        public void close_clutter(ArrayList _event)
        {
            if(onclose_clutter != null)
            {
                var argv0 = ((String)_event[0]);
                onclose_clutter( argv0);
            }
        }

	}
}

File: common/protcol/js/module/gm_centermodule.js
/*this module file is codegen by juggle for js*/
function gm_center_module(){
    eventobj.call(this);
    Imodule.call(this, "gm_center");

    this.confirm_gm = function(argv0){
        this.call_event("confirm_gm", [argv0]);
    }

    this.close_clutter = function(argv0){
        this.call_event("close_clutter", [argv0]);
    }

    this.close_zone = function(argv0, argv1){
        this.call_event("close_zone", [argv0, argv1]);
    }

    this.reload = function(argv0, argv1){
        this.call_event("reload", [argv0, argv1]);
    }

}
(function(){
    var Super = function(){};
    Super.prototype = Imodule.prototype;
    gm_center_module.prototype = new Super();
})();
gm_center_module.prototype.constructor = gm_center_module;

